content/departments/product-engineering/engineering/cloud/delivery/index.md

```diff
@@ -16,32 +16,56 @@ Enable any Sourcegraph customer or user to trial or run (in production) Sourcegr
 
-Specifically not included:
+## Contact
 
-- The release pipeline that serves sourcegraph.com - [Cloud Software-as-a-Service Team](../../cloud/devops/index.md)
+- [#delivery](https://sourcegraph.slack.com/archives/C02E4HE42BX) channel, or `@delivery-support` in Slack. Please follow our [support guidelines](#support-request-guidelines) below.
+- [team/delivery](https://github.com/sourcegraph/sourcegraph/labels/team%2Fdelivery) label and [@sourcegraph/delivery](https://github.com/orgs/sourcegraph/teams/delivery) team on GitHub.
 
-## Contact
+## Support
+
+If in doubt about the process, please ask in [#delivery](https://sourcegraph.slack.com/archives/C02E4HE42BX). Delivery teammates shold consult our [on-call rotation](./processes.md#on-call-rotation) guide to handle inquiries.
+
+### Requesting our support
+
+Feel free to direct simple questions to us in [#delivery](https://sourcegraph.slack.com/archives/C02E4HE42BX) in Slack.
 
-- [#delivery](https://sourcegraph.slack.com/archives/C02E4HE42BX) channel or @delivery-team in Slack.
-- [team/delivery](https://github.com/sourcegraph/sourcegraph/labels/team%2Fdelivery) label and @sourcegraph/delivery team on GitHub.
+- This channel _is_ regularly checked and well-monitored
+- So please do **NOT** directly message or CC an engineer—this is to try and protect their focus
+- Instead, if it’s urgent, please @ either the PM or the EM in the question in the channel and we'll ensure it gets the best response
 
-## Growth plan
+### Support request guidelines
+
+Support requests related to our [areas of ownership](index.md#responsibilities) should follow this process:
+
+1. Make sure there is an issue—if there's not, please create one and include:
+   - A short description of the ask
+   - A more detailed explanation of the background, the context and the challenge that needs solving
+   - Any guidance related to the impact this is having
+   - Any extra information that could help us solve or prioritize this
+2. Ensure lable `team/delivery` is added to the issue
+3. Ensure that the issue is added to the "[Delivery](https://github.com/orgs/sourcegraph/projects/205)" board in GitHub
+4. Anything without a status is checked and triaged weekly - so this is enough for feature requests or less urgent issues
+5. If you think this needs eyes 👀 sooner
+   - Within a few hours ➡️ message in [#delivery](https://sourcegraph.slack.com/archives/C02E4HE42BX)
+   - ASAP ➡️ message in [#delivery](https://sourcegraph.slack.com/archives/C02E4HE42BX) and CC `@delivery-support`
+
+<!-- ## Growth plan
 
 TODO
 
 ## Tech stack
 
-TODO
+TODO-->
 
 ## Principles
 
-We inherit Sourcegraph's [engineering principles and practices](../../process/principles-and-practices.md) and [Enablement org's principles and practices](../index.md#principles-and-practices).
+We inherit Sourcegraph's [engineering principles and practices](../../process/principles-and-practices.md).
```

Comment:
Delivery is not under Enablement anymore, and the cloud org page doesn't have this section.